clis/tcli/src/cli.rs

```diff
@@ -354,6 +356,17 @@ impl Cli {
         token: Option<String>,
         derp_server: Arc<Mutex<Option<Server>>>,
     ) -> anyhow::Result<Self> {
+        let (non_blocking_writer, _tracing_worker_guard) =
```

Comment:
Where was old subscriver inited, tcli main, now this is dublicated

Reply:
Old leftover creeped in during rebasing. Cleaned it up